ci/clean-up.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+source ci/header.sh
```

Comment:
I think you missed this one. Did you want to use the PATH based version here?

Reply:
Yes. I did miss it. Added it back in.